### PR TITLE
Cleans up CensusSyncTask documentation and marks as verified

### PR DESCRIPTION
**Pull request by desertaxle**
## Summary

The Census task has been tested against a live Census environment so we can mark the Census task as verified as part of our partnership with Census. 

As part of the verification I've also cleaned up the docs a little, but @tpdorsey please let me know if further improvement are needed.

## Checklist

This PR:

- [ ] adds new tests (if appropriate)
- [ ] adds a change file in the `changes/` directory (if appropriate)
- [ ] updates docstrings for any new functions or function arguments, including `docs/outline.toml` for API reference docs (if appropriate)

### CODE REVIEW

File: src/prefect/tasks/census/census.py [hunk 1]
```diff
@@ -19,8 +19,24 @@ class CensusSyncTask(Task):
     trigger section on the configuration page in the `api_trigger` param to set a default sync.
 
     Args:
-        - api_trigger (str, optional): default sync to trigger, if none is specified in `run`
+        - api_trigger (str, optional): default sync to trigger, if none is specified in `run`. The API
+            trigger URL for a sync can be found on sync's configuration page
+            (https://app.getcensus.com/syncs/{sync_id}/configuration) under Sync Triggers > API
```
Comment (tpdorsey) [inline src/prefect/tasks/census/census.py:24]:
```suggestion
            (https://app.getcensus.com/syncs/{sync_id}/configuration) under Sync Triggers > API.
```

File: src/prefect/tasks/census/census.py [hunk 2]
```diff
@@ -42,25 +58,26 @@ def run(
 
         Args:
             - api_trigger (str): if not specified in run, it will pull from the default for the
-                CensusSyncTask constructor. Keyword argument.
+                CensusSyncTask constructor. The API trigger URL for a sync can be found on sync's
+                configuration page (https://app.getcensus.com/syncs/{sync_id}/configuration) under
+                Sync Triggers > API
```
Comment (tpdorsey) [inline src/prefect/tasks/census/census.py:63]:
```suggestion
                Sync Triggers > API.
```

**Review by tpdorsey — DISMISSED** [review]
👍

File: src/prefect/tasks/census/census.py [hunk 3]
```diff
@@ -19,8 +19,24 @@ class CensusSyncTask(Task):
     trigger section on the configuration page in the `api_trigger` param to set a default sync.
 
     Args:
-        - api_trigger (str, optional): default sync to trigger, if none is specified in `run`
+        - api_trigger (str, optional): default sync to trigger, if none is specified in `run`. The API
```
Comment (tpdorsey) [inline src/prefect/tasks/census/census.py:22]:
```suggestion
        - api_trigger (str, optional): Default sync to trigger, if none is specified in `run`. The API
```

File: src/prefect/tasks/census/census.py [hunk 4]
```diff
@@ -42,25 +58,26 @@ def run(
 
         Args:
             - api_trigger (str): if not specified in run, it will pull from the default for the
-                CensusSyncTask constructor. Keyword argument.
+                CensusSyncTask constructor. The API trigger URL for a sync can be found on sync's
+                configuration page (https://app.getcensus.com/syncs/{sync_id}/configuration) under
+                Sync Triggers > API.
             - poll_status_every_n_seconds (int, optional): this task polls the Census API for the sync's
```
Comment (tpdorsey) [inline src/prefect/tasks/census/census.py:64]:
```suggestion
            - poll_status_every_n_seconds (int, optional): This task polls the Census API for the sync's
```

File: src/prefect/tasks/census/census.py [hunk 5]
```diff
@@ -42,25 +58,26 @@ def run(
...
-            {
-                'error_message': None / str,
-                'records_failed': int / None,
-                'records_invalid': int / None,
-                'records_processed': int / None,
-                'records_updated': int / None,
-                'status': 'completed'/'working'/'failed'
-            }
-            ```
+            - dict: dictionary of statistics returned by Census on the specified sync in
```
Comment (tpdorsey) [inline src/prefect/tasks/census/census.py:69]:
```suggestion
            - dict: Dictionary of statistics returned by Census on the specified sync in
```

File: src/prefect/tasks/census/census.py [hunk 6]
```diff
@@ -116,17 +133,17 @@ def run(
     @staticmethod
     def check_invalid_api(api_trigger: str):
         """
-        Makes sure the url for the api trigger matches the Census format specified below. If it does
-        not, it will raise a ValueError before returning.
+        Makes sure the url for the API trigger matches the Census format specified below. If it does
```
Comment (tpdorsey) [inline src/prefect/tasks/census/census.py:136]:
```suggestion
        Makes sure the URL for the API trigger matches the Census format specified below. If it does
```

File: src/prefect/tasks/census/census.py [hunk 7]
```diff
@@ -116,17 +133,17 @@ def run(
...
+        not, it will raise a ValueError.
 
         Format of api_trigger:
-            - "https://bearer:secret-token:s3cr3t@app.getcensus.com/api/v1/syncs/123/trigger"
+            - https://bearer:secret-token:{secret}@app.getcensus.com/api/v1/syncs/{sync_id}/trigger
 
         Args:
             - api_trigger (str): if specified in the constructor, will call this validation there
```
Comment (tpdorsey) [inline src/prefect/tasks/census/census.py:143]:
```suggestion
            - api_trigger (str): If specified in the constructor, will call this validation there
```

**Review by tpdorsey — DISMISSED** [review]
Looks good. Sorry if I missed some of these previously. Trying to move us to sentence casing and punctuation for args and returns.

**Review by madkinsz — APPROVED** [review]

**Pull request merged**